### ISSUE TITLE
Bug in the new Gale Level (Itch.io)

### ISSUE THREAD

**Issue by memeis** [bunfan/beat-banger-public#16]
At the beginning of Gale's level (10-20 seconds in) is one specific note which counts as hit but doesn't dissapear, resulting in one missed note.

**To Reproduce**
Steps to reproduce the behaviour:
Step 1: select Gale's level on 'virgin' difficulty
Step 2: try to hit every note

**Expected behavior**
The note should dissapear upon hitting it.

**Screenshots**
Sadly doesn't show anything that couldn't be achieved without a bug.

**Desktop (please complete the following information):**
 - OS: Windows 10 Home
 - Version: 2.9

**Additional context**
Basically, you can't get 5 stars with this.
